media/js/src/JXGBoard.jsx

```diff
@@ -404,42 +423,33 @@ export default class JXGBoard extends React.Component {
             }
         }
 
-        if (
-            (this.props.gType === 1 ||
-             this.props.gType === 10 ||
-             this.props.gType === 12 ||
-             this.props.gType === 14
-            ) && (
-                prevProps.gFunctionChoice !== this.props.gFunctionChoice ||
-                    prevProps.gCobbDouglasKName !== this.props.gCobbDouglasKName ||
-                    prevProps.gNName !== this.props.gNName
-            )
+        if ([1, 10, 12, 14, 24].includes(this.props.gType) && (
+            prevProps.gFunctionChoice !== this.props.gFunctionChoice ||
+            prevProps.gCobbDouglasKName !== this.props.gCobbDouglasKName ||
+            prevProps.gNName !== this.props.gNName
+        )
         ) {
-            const yLabel = getNLDSYLabel(
-                this.props.gType === 14 ? 1 : this.props.gFunctionChoice,
-                this.props.gCobbDouglasKName,
-                this.props.gNName
-            );
-            const xLabel = getNLDSXLabel(
-                this.props.gType === 14 ? 1 : this.props.gFunctionChoice,
-                this.props.gCobbDouglasKName,
-                this.props.gNName
-            );
-
             let board;
             if (this.board) {
                 board = this.board;
             }
 
-            if (this.board2 && (
-                this.props.gType === 12 || this.props.gType === 14
-            )) {
+            console.log(this.board2);
+            if (this.board2 && [12, 14, 24].includes(this.props.gType)
+            ) {
                 board = this.board2;
             }
 
             if (board.defaultAxes) {
-                board.defaultAxes.y.name = yLabel;
-                board.defaultAxes.x.name = xLabel;
+                if (this.props.gFunctionChoice === 0) {
```

Comment:
This new logic being introduced should only affect the new graph type, so I would surround this entire if/else in:
```
if (this.props.gType === 24) {
}
```

The other graph types which also may enter this if-statement include type 12 which does make use of gFunctionChoice and is having its bounding box updated by this code.

I know this is messy but I think just constraining this code to this graph type is sufficient for now - going forward I think we can figure out a cleaner way to cordon of this graph-specific bounding box code to the graph.